consensus/poet/core/sawtooth_poet/poet_consensus/poet_fork_resolver.py

```diff
@@ -58,5 +58,44 @@ def compare_forks(self, cur_fork_head, new_fork_head):
             Boolean: True if the new chain should replace the current chain.
             False if the new chain should be discarded.
         """
-        LOGGER.debug("PoetForkResolver.compare_forks()")
-        return new_fork_head.block_num > cur_fork_head.block_num
+        if new_fork_head.block_num > cur_fork_head.block_num:
+            LOGGER.info(
+                'Chain with new fork head %s...%s longer (%d) than current '
+                'chain head %s...%s (%d)',
+                new_fork_head.header_signature[:8],
+                new_fork_head.header_signature[-8:],
+                new_fork_head.block_num,
+                cur_fork_head.header_signature[:8],
+                cur_fork_head.header_signature[-8:],
+                cur_fork_head.block_num)
+            return True
+        elif new_fork_head.block_num < cur_fork_head.block_num:
+            LOGGER.info(
+                'Chain with current head %s...%s longer (%d) than new fork '
+                'head %s...%s (%d)',
+                cur_fork_head.header_signature[:8],
+                cur_fork_head.header_signature[-8:],
+                cur_fork_head.block_num,
+                new_fork_head.header_signature[:8],
+                new_fork_head.header_signature[-8:],
+                new_fork_head.block_num)
+            return False
+        elif new_fork_head.header_signature > cur_fork_head.header_signature:
+            LOGGER.info(
+                'Signature of new fork head (%s...%s) > than current '
+                '(%s...%s)',
+                new_fork_head.header_signature[:8],
+                new_fork_head.header_signature[-8:],
+                cur_fork_head.header_signature[:8],
+                cur_fork_head.header_signature[-8:])
+            return True
+
+        LOGGER.info(
```

Comment:
Might be good to put this in an `else` block, for clarity.  Seems a bit odd all by itself.